adaptive_scheduler/observation_portal_connections.py

```diff
@@ -83,12 +83,12 @@ def get_last_changed(self):
 
     @timeit
     @metric_timer('requestdb.get_requests', num_requests=len)
-    def get_all_request_groups(self, start, end, telescope_class='all'):
+    def get_all_request_groups(self, start, end, telescope_classes=[]):
```

Comment:
One more thing - Setting a default to a mutable value could cause some unexpected behavior.

Reply:
yup, codacy caught that too :)